mimic/rest/maas_api.py

```diff
@@ -631,6 +631,17 @@ def view_agent_host_info(self, request, tenant_id):
             "txnId": ".fake.mimic.transaction.id.c-1111111.ts-123444444.v-12344frf"
         })
 
+    @app.route('/v1.0/<string:tenant_id>/agent_installers', methods=['POST'])
+    def agent_installer(self, request, tenant_id):
+        """
+        URL of agent install script
+        """
+        xsil = "https://monitoring.api.rackspacecloud.com/"
+        xsil += "v1.0/747440/agent_installers/c69b2ceafc0444506fb32255af3d9be3.sh"
```

Comment:
can we change to be the mimic url given in the mimic service catalog appended by the hash.